website/content/en/docs/Configure/standard_to_managed_mode.md

```diff
@@ -10,7 +10,7 @@ description: >
 
 ## Introduction
 
-AAD Pod Identity supports 2 modes of operation:
+AAD Pod Identity supports 2 modes of operation ([full explanation of both modes](https://learn.microsoft.com/en-us/azure/aks/use-azure-ad-pod-identity#operation-mode-options)):
```

Comment:
can we link to this page? https://azure.github.io/aad-pod-identity/docs/configure/pod_identity_in_managed_mode/

Reply:
Yes! Updated-